scripts/ci/create-docker-image.sh

```diff
@@ -29,9 +30,28 @@ GOBUILD_DIR=/root/.cache/go-build
 TOPLEVEL_VOL=$PWD
 TOPLEVEL_DIR=/root/go/src/github.com/skydive-project/skydive
 
+LAST_VERSION=$( git describe --abbrev=0 --tags | sed -e s/v// )
+LAST_COMMIT=$( git rev-parse --short=12 HEAD )
```

Comment:
This doesn't work as Jenkins creates a merge commit before testing the PR. You can `REF` as in the other tests.

Reply:
@lebauce - `REF=latest` in the relevant case - can I not assume that Jenkins does a `git checkout ..` before invoking this script (and then `HEAD` points to the tip).